reviews-receiver/.prettierrc.json

```
@@ -0,0 +1,7 @@
+{
+    "tabWidth": 2,
+    "semi": false,
```

for prettier consistent across all services
```suggestion
    "semi": true,
```